### DISCUSSION TITLE
Is it possible to have several http_server sources with the same port but different paths?

### DISCUSSION THREAD

**Discussion by gl1ridae** [vectordotdev/vector#18498] [Q&A] [answered]
Hi all! The question is more or less in the title. I wonder if it's possible with 2 or more http_server sources, all listening same port but different paths? Can figure it out myself.
Thank you in advance!

**Comment by jszwedko** [comment, marked as answer]
Hi @gl1ridae ! Not at this time. The `http_server` source binds to the port so if you have two they will both try to bind to the same port. You could instead use a `route` transform after the `http_server` source to handle events differently based on the path they were sent to. You'll want to set https://vector.dev/docs/reference/configuration/sources/http_server/#strict_path to `false` to allow events on any path.

Otherwise you'll unfortunately need to use different ports.

**Reply by gl1ridae** [reply]
Hi @jszwedko! Thank you for clarification! Got it now, totally missed the strict_path option.
